### PR TITLE
[FR] Clean up How to time songs

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for cleanup the French translation of the wiki page `Guides\How_to_Time_Songs`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Guides/How_to_Time_Songs/fr.md [hunk 1]
```diff
@@ -128,7 +122,7 @@ Si tout a été fait correctement, vous devriez maintenant avoir une beatmap cor
...
 
-Vous pouvez demander de l'aide pour cela, mais il n'y a que très peu de personnes qui *savent* comment identifier et travailler avec du multitiming. De plus, personne ne va vous aider si vous ne mettez pas de réels efforts dans vos beatmaps ou si ils ne pensent pas que vous valez leur temps. Vous pouvez le faire vous-même si vous avez beaucoup d'expérience et/ou une très bonne précision, et que vous êtes capable de toucher les notes correctement. Ceci est vital, car vous allez avoir besoin de faire beaucoup de tests, et des parties mal timées vont être frustrantes pour le joueur.
+Vous pouvez demander de l'aide pour cela, mais il n'y a que très peu de personnes qui *savent* comment identifier et travailler avec du multitiming. De plus, personne ne va vous aider si vous ne mettez pas de réels efforts dans vos beatmaps ou si ils ne pensent pas que vous valez leur temps. Vous pouvez le faire vous-même si vous avez beaucoup d'expérience et/ou une très bonne précision, et que vous êtes capable de frapper les notes correctement. Ceci est vital, car vous allez avoir besoin de faire beaucoup de tests, et des parties mal timées vont être frustrantes pour le joueur.
```
Comment (Ganondorf-1) [inline wiki/Guides/How_to_Time_Songs/fr.md:125]:
```suggestion
Vous pouvez demander de l'aide pour cela, mais il n'y a que très peu de personnes qui *savent* comment identifier et travailler avec du multitiming. De plus, personne ne va vous aider si vous ne mettez pas de réels efforts dans vos beatmaps ou si ils ne pensent pas que vous valez leur temps. Vous pouvez le faire vous-même si vous avez beaucoup d'expérience et/ou une très bonne précision, et que vous êtes capable de toucher les notes correctement. Ceci est vital, car vous allez avoir besoin de faire beaucoup de tests, et des parties mal timées vont être frustrantes pour le joueur.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Guides/How_to_Time_Songs/fr.md [hunk 2]
```diff
@@ -168,7 +162,7 @@ Au bout du compte, vous devriez retenir ces quelques conseils importants :
 - **Gardez toujours un œil sur les résultats des tests.**
 - **Appuyez vous aussi sur vos autre sens, n'utilisez pas que vos oreilles.** Parfois, la même partie d'une musique peut-être perçue différemment. Essayez de vous concentrer sur les résultats des tests, pas seulement sur votre audition.
 - **Les hit errors de moins de 3 ms, ne veulent rien dire et peuvent être ignorés.** Vous devriez commencer à les prendre en compte quand ils commencent à monter vers 5-6 millisecondes ou plus.
-- **Have a break. Have a Kit-Kat. (Faites des pauses.)** Conseil du docteur.
+- **Have a break. Have a Kit-Kat. (Faites des pauses).** Conseil du docteur.
```
Comment (Walavouchey) [inline wiki/Guides/How_to_Time_Songs/fr.md:165]: if parentheses encompass a whole sentence, the period goes inside

```suggestion
- **Have a break. Have a Kit-Kat. (Faites des pauses.)** Conseil du docteur.
```

File: wiki/Guides/How_to_Time_Songs/fr.md [hunk 3]
```diff
@@ -66,7 +60,7 @@ Maintenant que l'offset est correct, vous pouvez ajuster le BPM. Déterminer l'o
...
+Pour regarder si le BPM est correct ou non, allez directement vers le milieu ou la fin de la musique, là où la différence entre le tempo de la musique et celui du métronome est accumulée est facilement audible. Souvent, le BPM va être un nombre entier (pas de valeurs décimales) donc essayez d'ajuster le BPM jusqu'au nombre entier le plus proche pour voir si cela fonctionne dans votre cas. Après cela, vous pouvez essayer de faire des ajustements plus précis comme 101.200 au lieu de 101BPM, voir éventuellement 101.238, pour nommer quelques exemples. Avant de laisser le BPM avec des valeurs dans les centièmes (.01) ou dizaines (.1), enlevez les décimales ou arrondissez les puis réécoutez le métronome. Si la différence est insignifiante ou si vous la trouvez plus précise (regardez le milieu et la fin encore une fois), vous pouvez le garder et considérer le problème comme réglé. Attention : Toutes les musiques électroniques n'ont pas de BPM complètement plat, en dépit de quel point elle peuvent sembler banales. Enlever les décimales peut résulter en une erreur de BPM, dans ce cas-là.
```
Comment (Walavouchey) [inline wiki/Guides/How_to_Time_Songs/fr.md:63]: not sure if ".1" is specific to english, i've at least never seen it in any other language

```suggestion
Pour regarder si le BPM est correct ou non, allez directement vers le milieu ou la fin de la musique, là où la différence entre le tempo de la musique et celui du métronome est accumulée est facilement audible. Souvent, le BPM va être un nombre entier (pas de valeurs décimales) donc essayez d'ajuster le BPM jusqu'au nombre entier le plus proche pour voir si cela fonctionne dans votre cas. Après cela, vous pouvez essayer de faire des ajustements plus précis comme 101,200 au lieu de 101 BPM, voir éventuellement 101,238, pour nommer quelques exemples. Avant de laisser le BPM avec des valeurs dans les centièmes (0,01) ou dizaines (0,1), enlevez les décimales ou arrondissez les puis réécoutez le métronome. Si la différence est insignifiante ou si vous la trouvez plus précise (regardez le milieu et la fin encore une fois), vous pouvez le garder et considérer le problème comme réglé. Attention : Toutes les musiques électroniques n'ont pas de BPM complètement plat, en dépit de quel point elle peuvent sembler banales. Enlever les décimales peut résulter en une erreur de BPM, dans ce cas-là.
```

File: wiki/Guides/How_to_Time_Songs/fr.md [hunk 4]
```diff
@@ -16,9 +10,9 @@ Vous êtes aussi supposé savoir ce qu'est le [BPM](/wiki/Beatmapping/Beats_per_
...
+Si vous allez travailler sur une beatmap et que vous comptez la rendre classée (ou approuvée), il est nécessaire pour le fichier audio d'avoir un bitrate entre 128 et 192kbps. (Ceci est pour s'assurer que les musiques sont d'une qualité audio acceptable et ne prennent pas trop de place sur les serveurs ou sur votre disque dur). Vous pouvez voir le bitrate d'un fichier audio en faisant un clic droit sur votre fichier et en sélectionnant Propriétés, puis la fenêtre Détails. Si vous ne savez pas comment réduire le bitrate (d'habitude jusqu'à 192kbps), il existe le [guide de l'édition audio](/wiki/Guides/Audio_Editing) ainsi [qu'une leçon de la part de notre propre professeur](https://www.youtube.com/watch?v=muu3HkG38kk). Vous pouvez demander à vos amis ou dans le tchat comment faire. Si rien de cela ne fonctionne, cherchez/demandez sur les [forums](https://osu.ppy.sh/community/forums/56) ou trouvez par vous-même comment faire.
```
Comment (Walavouchey) [inline wiki/Guides/How_to_Time_Songs/fr.md:13]:
```suggestion
Si vous allez travailler sur une beatmap et que vous comptez la rendre classée (ou approuvée), il est nécessaire pour le fichier audio d'avoir un bitrate entre 128 et 192 kbps. (Ceci est pour s'assurer que les musiques sont d'une qualité audio acceptable et ne prennent pas trop de place sur les serveurs ou sur votre disque dur). Vous pouvez voir le bitrate d'un fichier audio en faisant un clic droit sur votre fichier et en sélectionnant Propriétés, puis la fenêtre Détails. Si vous ne savez pas comment réduire le bitrate (d'habitude jusqu'à 192 kbps), il existe le [guide de l'édition audio](/wiki/Guides/Audio_Editing) ainsi [qu'une leçon de la part de notre propre professeur](https://www.youtube.com/watch?v=muu3HkG38kk). Vous pouvez demander à vos amis ou dans le tchat comment faire. Si rien de cela ne fonctionne, cherchez/demandez sur les [forums](https://osu.ppy.sh/community/forums/56) ou trouvez par vous-même comment faire.
```

**Review by cl8n — APPROVED** [review]

**Pull request merged**